configs/VEkTORF722/config.h

```diff
@@ -0,0 +1,110 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU   STM32F7X2
+#define BOARD_NAME      VEkTORF722
+#define MANUFACTURER_ID PYI
```

Comment:
Unknown manufacturer. Needs to be 4 characters and be added to manufacturers.md
```suggestion
#define MANUFACTURER_ID PYI
```

Reply:
the manufacturer is named "PYI" - what could the 4th character be?

Reply:
26 posibilities :)

Reply:
Neither fit into the manufacturers name